test/common/api/os_sys_calls_test.cc

```diff
@@ -1,9 +1,65 @@
+#include <fcntl.h>
```

Comment:
do you need this? Aren't we working entirely with the OSSysCalls abstraction?

Reply:
Yeah, it won't build on Windows without it because `O_CREAT` and `O_READWR` aren't defined without that header.